.github/workflows/ci.yml

```diff
@@ -380,13 +380,15 @@ jobs:
         -DCMAKE_PREFIX_PATH:PATH=$(pwd)/../dependencies/install/
         -DF3D_COVERAGE=ON
         -DF3D_MODULE_EXTERNAL_RENDERING=ON
+        -DF3D_MODULE_EXR=ON
```

Comment:
rebase on master ? this should not be here

Reply:
Already rebased on master, it was missing so I think nothing about EXR was actually tested for coverage.

Reply:
it is: https://github.com/f3d-app/f3d/blob/e3d0dd398c9f4f5bec1f935915248e8e17170e7b/.github/workflows/ci.yml#L170

Reply:
line 383 vs line 170 🤨

Reply:
The variable is enabled for the regular CI, but it wasn't enabled for the coverage.

Reply:
right!